### PR TITLE
Climate summit

### PR DESCRIPTION
**Pull request by dialite**
# For this project, I did the following:

- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements in the page.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.
- Communicate technical concepts to other technical people.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+          <a href="#"><i class="fa fa-mastodon"></i></a>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </div>
+      </div>
+
+      <!-- Navbar with sub-titles link -->
+      <div id="nav-bar">
+        <a href="#" id="logo">COP27<span>2022</span></a>
+        <nav id="menu">
+          <span id="menu-close-btn">×</span>
```
Comment (tufoinnkuo10) [inline index.html:35]: - [ ] please your hamburger icon does not work on the about page of the mobile version. please kindly check that
- [ ] kindly ensure that background of your menu is transparent or the same color/pattern as the top section background.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+          <a href="#" class="home-link">Home</a>
+          <a href="#program">Program</a>
+          <a href="#speakers">Speakers</a>
```
Comment (tufoinnkuo10) [inline index.html:38]: - [ ] Great job. you are missing the top short bar section with social media links, logout, and language links aligned to the right
Screenshot (541)_LI

File: index.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        </div>
+        <div class="item">
+          <img src="img/icons-3.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Awards</h3>
+          <p class="item-description">
+            Awards
+          </p>
+        </div>
+        <div class="item">
+          <img src="img/icons-4.svg" class="item-icon" alt="" />
```
Comment (tufoinnkuo10) [inline index.html:104]: - [ ] please kindly use five articles like in the behance design
yours
Screenshot (542)
behance
Screenshot (544)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+      <h2 class="section-heading">Partners</h2>
+      <div class="sponsors-list">
+        <img
+          src="img/climate-partners.png"
+          title="sabmiller"
+          alt="sabmiller"
+        />
+      </div>
+    </section>
+
```
Comment (tufoinnkuo10) [inline index.html:156]: - [ ] Great design. please kindly use two photos as used in the original design.
yours
Screenshot (545)
behance
Screenshot (546)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+    <!-- Title welcome page section -->
+    <section id="section1" class="container">
+      <div class="wrapper">
+        <img src="img/bg-bob.png" class="shape-set" alt="" />
+        <h1>World Climate Summit - COP27</h1>
+        <h4>The Investment COP</h4>
+        <h2>"Sustainable Environment"</h2>
+        <p class="description">
+          World Climate Summit -
+          The Investment COP will utilise a decade of experience to bring together
```
Comment (tufoinnkuo10) [inline index.html:67]: - [ ] please ensure that your partners are showing beside the footer of the about page
- [ ] also, please follow the original design and make the footer  of the about page mobile version `white` color
yours
Screenshot (548)
behance
Screenshot (547)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @dialite 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


## Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md [hunk 6]
```diff
@@ -1,2 +1,62 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Cindy Shin called CC Global Summit 2015 - Design Initiator
+- Font Awesome
```
Comment (christianbiring1) [inline README.md:57]: [OPTIONAL]
- [ ] Kindly make sure you give the appropriate credit to Cindy and why not provide the `link` to his/her Behance account :smile:

File: index.html (hunk 1 above)
Comment (christianbiring1) [reply index.html:35]: fixed :+1:

File: index.html (hunk 2 above)
Comment (christianbiring1) [reply index.html:38]: Fixed :+1:

File: index.html (hunk 5 above)
Comment (christianbiring1) [reply index.html:67]: Great job implementing the required changes :smile: :+1:

File: index.html (hunk 3 above)
Comment (christianbiring1) [reply index.html:104]: Fixed :+1: 
[OPTIONAL]
- [ ] However, You may consider surrounding your card with a white square when it is hovered as in the design :smile: 
- [ ] Also the Participate is not supposed to have that style. As you can see in the template. It is only a simple white link with a text-decoration :smile:

**Review by christianbiring1 — APPROVED** [review]
Hi @dialite , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
congraaaa

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good looking UI :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**